Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/OneTimePassword.kt

```diff
@@ -0,0 +1,15 @@
+package de.rki.coronawarnapp.datadonation
+
+import android.util.Base64
+import android.util.Base64.encode
+import java.nio.ByteBuffer
+import java.util.UUID
+
+data class OneTimePassword(val uuid: UUID = UUID.randomUUID()) {
+
+    val payloadForRequest: ByteArray
+        get() = ByteBuffer.wrap(ByteArray(16)).apply {
```

Comment:
Is this different from `UUID.toString().toByteArray()` ?

Reply:
technically it's not the same, but since we're just in need of a random byte array, and don't want to convert back to uuid, i guess your suggestion will suffice. will change it